GitExtensions/Program.cs

```diff
@@ -167,6 +168,16 @@ private static string GetWorkingDir(string[] args)
                 }
             }
 
+            if (workingDir == "")
+            {
+                var cwd = Environment.CurrentDirectory;
+                var gitDir = GitModule.FindGitWorkingDir(cwd);
+                if (GitModule.IsValidGitWorkingDir(gitDir))
```

Comment:
not related to the PR, but a general observation - this API is very strange - first we ask fro a working dir, then we double check it is a working dir indeed...